### DISCUSSION TITLE
Should'nt we specify `botocore>=1.0.0` in `pyproject.toml` ?

### DISCUSSION THREAD

**Discussion by wenceslas-sanchez** [encode/httpx#2930] [Potential Issue] [answered]
With the new version of httpx `0.25.1`, if we did have an old version of httpcore (like `0.18.x`), the update of httpx do not update httpcore and then httpx `0.25.1` doesn't work because of httpcore (issue regarding the instanciation `httpx.Transport()`); shouldn't you set the httpcore version above `1.0.0` to be sure httpx install a httpcore version that actually works with `0.25.1` ?

**Comment by lovelydinosaur** [comment, marked as answer]
Ah right, I wasn't aware of our lower bound here.
Yep that's a good call.

`httpcore==1.*` would be a sensible requirement in the pyproject.toml.
